packages/jwk-did-provider/__tests__/jwk-did-resolver.test.ts

```diff
@@ -0,0 +1,99 @@
+import { createAgent, DIDResolutionResult, IKeyManager } from '@veramo/core'
+import { DIDManager, MemoryDIDStore } from '@veramo/did-manager'
+import { DIDResolverPlugin } from '@veramo/did-resolver'
+import { KeyManager, MemoryKeyStore, MemoryPrivateKeyStore } from '@veramo/key-manager'
+import { KeyManagementSystem } from '@veramo/kms-local'
+import { Resolver } from 'did-resolver'
+import { getDidJwkResolver, JwkDIDProvider } from '../src'
+
+const DID_METHOD = 'did:jwk'
+
+const jwkDIDProvider = new JwkDIDProvider({
+  defaultKms: 'mem',
+})
+
+const agent = createAgent<IKeyManager, DIDManager>({
+  plugins: [
+    new DIDResolverPlugin({
+      resolver: new Resolver({...getDidJwkResolver()})
+    }),
+    new KeyManager({
+      store: new MemoryKeyStore(),
+      kms: {
+        mem: new KeyManagementSystem(new MemoryPrivateKeyStore()),
+      },
+    }),
+    new DIDManager({
+      providers: {
+        [DID_METHOD]: jwkDIDProvider,
+      },
+      defaultProvider: DID_METHOD,
+      store: new MemoryDIDStore(),
+    }),
+  ],
+})
+
+describe('@sphereon/jwk-did-resolver', () => {
+  it('should resolve did:jwk', async () => {
+    const didResolutionResult: DIDResolutionResult = await agent.resolveDid({ didUrl: 'did:jwk:eyJrdHkiOiJFQyIsImNydiI6InNlY3AyNTZrMSIsIngiOiJCS2NlRjMwbHBTNkptT1RsS09LQVdudGtKdVRCSzNGX1JoaXlEcTRtdm9jIiwieSI6Im9WY1phQnpiSFJ2UW5iSXhwRWRXbVlRMGtSRm42ajVDRkVQcGxvX09ON1UifQ' })
+
+    expect(didResolutionResult.didResolutionMetadata).toBeDefined()
```

Comment:
Could you check against exact values in this class? 
We then get to catch any regressions.

Reply:
changed